// flatpickr/dist/l10n/lt.d 2.ts
import { CustomLocale } from "../types/locale";
export declare const Lithuanian: CustomLocale;
declare const _default: {
    default?: CustomLocale | undefined;
    hr?: CustomLocale | undefined;
    th?: CustomLocale | undefined;
    tr?: CustomLocale | undefined;
    ar?: CustomLocale | undefined;
    at?: CustomLocale | undefined;
    az?: CustomLocale | undefined;
    be?: CustomLocale | undefined;
    bg?: CustomLocale | undefined;
    bn?: CustomLocale | undefined;
    bs?: CustomLocale | undefined;
    cat?: CustomLocale | undefined;
    cs?: CustomLocale | undefined;
    cy?: CustomLocale | undefined;
    da?: CustomLocale | undefined;
    de?: CustomLocale | undefined;
    en?: CustomLocale | undefined;
    eo?: CustomLocale | undefined;
    es?: CustomLocale | undefined;
    et?: CustomLocale | undefined;
    fa?: CustomLocale | undefined;
    fi?: CustomLocale | undefined;
    fo?: CustomLocale | undefined;
    fr?: CustomLocale | undefined;
    gr?: CustomLocale | undefined;
    he?: CustomLocale
